File: JavaScript/8js 2/human_cd.js
/*
|--------------------------------------------------------------------------
| Cat years, Dog years
|--------------------------------------------------------------------------
|
*/

function humanYearsCatYearsDogYears(x) {
   return [
      x,
      x == 1 ? 15 : x == 2 ? 24 : x > 2 ? 24 + (x - 2) * 4 : 0,
      x == 1 ? 15 : x == 2 ? 24 : x > 2 ? 24 + (x - 2) * 5 : 0,
   ];
}


function humanYearsCatYearsDogYears(x) {
   return [x, (x - 1 ? 16 : 11) + 4 * x, (x - 1 ? 14 : 10) + 5 * x];
}


function humanYearsCatYearsDogYears(x) {
   switch (x) {
      case 1:
         return [1, 15, 15];
      case 2:
         return [2, 24, 24];
      default:
         return [x, (x - 2) * 4 + 24, (x - 2) * 5 + 24];
   }
}


console.log(humanYearsCatYearsDogYears(1), [1, 15, 15]);
console.log(humanYearsCatYearsDogYears(2), [2, 24, 24]);
console.log(humanYearsCatYearsDogYears(10), [10, 56, 64]);

File: Python/8py 2/human_cd.py
"""
|--------------------------------------------------------------------------
| Cat years, Dog years
|--------------------------------------------------------------------------
|
"""

def human_years_cat_years_dog_years(x):
   return [
      x,
      15 if x == 1 else 24 if x == 2 else 24 + (x - 2) * 4 if x > 2 else 0,
      15 if x == 1 else 24 if x == 2 else 24 + (x - 2) * 5 if x > 2 else 0
   ]


def human_years_cat_years_dog_years(n):
   return [n, 15 if n == 1 else 4 * (n + 4), 15 if n == 1 else 5 * n + 14]


def human_years_cat_years_dog_years(x):
   return [x, 16 + 4 * x, 14 + 5 * x] if x > 1 else [1,15,15]


print(human_years_cat_years_dog_years(1), [1,15,15])
print(human_years_cat_years_dog_years(2), [2,24,24])
print(human_years_cat_years_dog_years(10), [10,56,64])